.github/workflows/build-blst.bash

```diff
@@ -7,4 +7,20 @@ git clone https://github.com/supranational/blst
 cd blst
 git reset --hard $BLST_REF
 ./build.sh
+
+mkdir -p pkgconfig
```

Comment:
separate question: should we try and upstream this to the blst repo? they might like it

Reply:
... are we going to have to put this in every single downstream project's GHA workflows? :(

Reply:
Yea, no. This is going to suck too much. We need to have a better solution. But let's make this work first.

Reply:
We can later create a github action, similar to how secp256k1 is handled everywhere: https://github.com/input-output-hk/setup-secp256k1

Reply:
It won't be necessary for libsodium, in fact I would discourage using custom libsodium on GA CI, because we promise our crypto libs to work with stock libsodium, so it is good that we test that version as well.